django_fido/views.py

```diff
@@ -46,10 +47,13 @@ class Fido2ViewMixin(object):
         If None, the RP ID is used instead.
     @cvar attestation: Attestation conveyance preference,
                        see https://www.w3.org/TR/webauthn/#enumdef-attestationconveyancepreference
+    @cvar attestation_types: Allowed attestation format types.
+        If None, all attestation formats except `none` are allowed.
     @cvar session_key: Session key where the FIDO 2 state is stored.
     """
 
     attestation = AttestationConveyancePreference.NONE
+    attestation_types = None  # type: Optional[Attestation]
```

Comment:
Shouldn't this be `Optional[List[Attestation]]`?

Reply:
It should.